app/models/user.rb

```diff
@@ -121,6 +121,16 @@ def can_access_user_profile?
     loa1? || loa2? || loa3?
   end
 
+  # User's profile contains a list of VHA facility-specific identifiers.
+  # Facilities in the defined range are treating facilities, indicating
+  # that the user is a VA patient.
+  def va_patient?
```

Comment:
Moved out of `mhv_account` (where it was private) so that it can be accessed by both `mhv_account` and `mhv_accounts_service` through `user.va_patient?`. Makes more sense to me than `user.mhv_account.va_patient?`.